### PR TITLE
Added allowDeselect to multiple select

### PR DESCRIPTION
**Pull request by dminkovski**
Issue: https://github.com/brianvoe/slim-select/issues/457

Please let me know if this approach is ok or you want me to change something 👍 
Thanks for your support and guidance!

### CODE REVIEW

File: src/slim-select/store.ts [hunk 1]
```diff
@@ -338,4 +338,8 @@ export default class Store {
 
     return dataSearch
   }
+
+  public getSelectType(): string {
```
Comment (dminkovski) [inline src/slim-select/store.ts:342]: Added this for testing purposes mainly.

File: src/slim-select/index.test.ts [hunk 2]
```diff
@@ -0,0 +1,99 @@
+/**
```
Comment (brianvoe) [inline src/slim-select/index.test.ts:1]: I love that you were able to add tests. I hate front end testing with a passion.

File: src/slim-select/store.ts (hunk 1 above)
Comment (brianvoe) [reply src/slim-select/store.ts:342]: totally fine with me

**Comment by brianvoe** [comment]
This looks good to me. Ill merge this in and test things and see if everything checks out.

If your up for it tests would be something I would love to get in and show up on github actions.

**Pull request merged**